src/v0/destinations/facebook_pixel/transform.test.js

```diff
@@ -0,0 +1,68 @@
+const { handleSearch } = require('../../../../src/v0/destinations/facebook_pixel/transform');
+
+describe('Unit test cases for facebook_pixel handle search', () => {
+  let message = {
```

Comment:
is it better to make this function instead? Like getSampleMessage() so that each test case has different object

Reply:
done